dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/ApacheDSandKDC.java

```diff
@@ -653,17 +653,44 @@ public static void createKeyTabFile() throws Exception {
 
         Log.info(c, "createKeyTabFile", "Created keytab: " + keytabFile);
         Log.info(c, "createKeyTabFile", "Keytab actual contents: " + FileUtils.readFile(keytabFile));
+    }
+
+    /**
+     * @param entries
+     */
+    protected static List<KeytabEntry> addKerberosKeysToKeytab(String principalName, String principalPwd) {
```

Comment:
This seems misnamed. Seems to create a new list of KeytabEntry's for the principalName and principalPwd.